### PR TITLE
Socomecpdu support

### PR DESCRIPTION
**Pull request by Negatifff**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
> Please add test data so we can ensure your change is not broken in the future.
> 
> Read the docs to find out how: https://docs.librenms.org/Developing/os/Test-Units

Missing step 2

**Comment by Negatifff** [comment]
> > Please add test data so we can ensure your change is not broken in the future.
> > Read the docs to find out how: https://docs.librenms.org/Developing/os/Test-Units
> 
> Missing step 2

sorry, it's new for me.
I hope now I did everything correct

File: includes/definitions/discovery/socomec.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+mib: SOCOMECPDU-MIB
+modules:
+    os:
+        features: SOCOMECPDU-MIB::dpduIdentModel.0
```
Comment (Jellyfrog) [inline includes/definitions/discovery/socomec.yaml:4]: Seems this Oid should be the hardware instead?

Comment (Negatifff) [reply includes/definitions/discovery/socomec.yaml:4]: I was not sure what to specify here. used as an example panduit.yaml - there was a branch with a model.

Comment (Jellyfrog) [reply includes/definitions/discovery/socomec.yaml:4]: If you look at the data it says;
"hardware": "00",
"features": "NRT-OP-PDU1-28"

So hardware seems to just be a revision, not so interesting, and the feature is the model

Comment (Negatifff) [reply includes/definitions/discovery/socomec.yaml]: tried to fix it.

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**